tests/codegen/slice-pointer-nonnull-unwrap.rs

```diff
@@ -0,0 +1,19 @@
+//@ compile-flags: -O
+//@ min-llvm-version: 18
+#![crate_type = "lib"]
+
+use std::ptr::NonNull;
+
+// CHECK-LABEL: @slice_ptr_len_1
+// CHECK: {{.*}}:
```

Comment:
```suggestion
// CHECK-NEXT: {{.*}}:
```
If you're matching the BB label, you should be able to use NEXT I think?